code/src/WijmoProvider/Features/Export.ts

```diff
@@ -45,10 +57,38 @@ namespace WijmoProvider.Feature {
             this._grid.features.pagination.changePageSize(0);
         }
 
+        private _showLoadingMessage() {
+            const gridId = this._grid.widgetId;
+            const parentPlaceholder = document.querySelector(
+                '#' +
+                    gridId +
+                    ' .' +
+                    OSFramework.Helper.Constants.gridAutogeneratedPlaceholderCss
+            );
+            const loadingPlaceholderContent = `<div class="datagrid-loading heading4 OSInline" style="width: 100%; height: 100%;"><i class="icon fa-spin fa fa-spinner fa-1x"></i><div class="OSInline" style="margin-left: 10px;">${this._loadingMessage}</div></div>`;
+            const createdDivElem = document.createElement('div');
+            createdDivElem.className =
+                OSFramework.Helper.Constants.overlayExportFeedbackCss;
+            createdDivElem.innerHTML = loadingPlaceholderContent;
+            parentPlaceholder.appendChild(createdDivElem);
+        }
+
         public build(): void {
             return;
         }
 
+        public customizeExportingMessage(
+            exportingMessage: string,
+            showMessage = true
+        ): void {
+            if (exportingMessage !== '' && exportingMessage !== undefined) {
+                this._loadingMessage =
+                    OSFramework.Helper.Sanitize(exportingMessage);
```

Comment:
If we were to use the suggestion above, this would no longer be needed. But I would leave as is, just to be sure :)